.pipelines/e2e-aks-arc.yaml

```diff
@@ -8,15 +13,35 @@ jobs:
     - template: templates/install-oras.yaml
     - template: templates/install-bats.yaml
     - template: templates/package-helm-chart.yaml
-    - template: templates/publish-helm-chart.yaml
+    - ${{ if eq(parameters.releaseE2E, true) }}:
```

Comment:
Can you collapse this into one line where you pass through parameters.releaseE2E for releaseTag?

Reply:
good catch